### PR TITLE
Add Extra Mappings

### PR DESCRIPTION
**Pull request by tl0x**

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -171,6 +178,7 @@ CLASS net/minecraft/class_39 net/minecraft/block/Block
 		ARG 3 y
 		ARG 4 z
 		ARG 5 face
+	METHOD method_121 returnZero ()I
```
Comment (sschr15) [inline mappings/net/minecraft/block/Block.mapping]: Is this just a method with a body of `return 0;`? If so, maybe see where it is used. If it's not used anywhere, this name could be acceptable, but otherwise I think it should be changed.

File: mappings/net/minecraft/block/Flower.mapping [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
 	METHOD method_127 canPlace (I)Z
 		ARG 1 blockId
-	METHOD method_128 (Lnet/minecraft/class_101;III)Z
+	METHOD method_128 canPlaceFlower (Lnet/minecraft/class_101;III)Z
 		ARG 1 world
-	METHOD method_129 (Lnet/minecraft/class_101;III)V
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_129 checkShouldbeBroken (Lnet/minecraft/class_101;III)V
```
Comment (sschr15) [inline mappings/net/minecraft/block/Flower.mapping]: This seems a bit verbose. 
```suggestion
	METHOD method_129 shouldBreak (Lnet/minecraft/class_101;III)V
```

File: mappings/net/minecraft/block/TransparentBlock.mapping [hunk 3]
```diff
@@ -1,3 +1,7 @@
 CLASS net/minecraft/class_46 net/minecraft/block/TransparentBlock
+	FIELD field_178 transparent Z
```
Comment (sschr15) [inline mappings/net/minecraft/block/TransparentBlock.mapping]: Is this public? If not, does it have an accessor? In either case, the public-facing member possibly should be prefixed with `is` in traditional Java style

File: mappings/net/minecraft/client/entity/particle/Particle.mapping [hunk 4]
```diff
@@ -15,6 +15,7 @@ CLASS net/minecraft/class_262 net/minecraft/client/entity/particle/Particle
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
-	METHOD method_650 (Lnet/minecraft/class_188;FFFFFF)V
+	METHOD method_650 render (Lnet/minecraft/class_188;FFFFFF)V
 		ARG 1 tesselator
+		ARG 2 g
```
Comment (sschr15) [inline mappings/net/minecraft/client/entity/particle/Particle.mapping]: What is `g`? I'm personally curious

File: mappings/net/minecraft/entity/Entity.mapping [hunk 5]
```diff
@@ -70,6 +77,8 @@ CLASS net/minecraft/class_140 net/minecraft/entity/Entity
 		ARG 1 tag
 	METHOD method_350 deserializeDefaultData (Lnet/minecraft/class_5;)V
 		ARG 1 tag
+	METHOD method_352 initPlayer ()V
+	METHOD method_353 setDead ()V
```
Comment (sschr15) [inline mappings/net/minecraft/entity/Entity.mapping]: Influenced by MCP, I'd presume? Maybe something more informative could work, possibly like this? 
```suggestion
	METHOD method_353 markForRemoval ()V
```

File: mappings/net/minecraft/entity/living/LivingEntity.mapping [hunk 6]
```diff
@@ -7,15 +7,21 @@ CLASS net/minecraft/class_154 net/minecraft/entity/living/LivingEntity
...
+		ARG 2 y
+		ARG 3 z
 	METHOD method_390 heal (I)V
 		ARG 1 amount
 	METHOD method_391 getDropItem ()I
 	METHOD method_392 updateMovements ()V
 	METHOD method_393 onDeath (Lnet/minecraft/class_140;)V
 		ARG 1 killer
 	METHOD method_394 livingTick ()V
+	METHOD method_395 returnNullString ()Ljava/lang/String;
```
Comment (sschr15) [inline mappings/net/minecraft/entity/living/LivingEntity.mapping]: See my previous thoughts on `returnZero ()I`

File: mappings/net/minecraft/item/Seeds.mapping [hunk 7]
```diff
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_119 net/minecraft/item/Seeds
 	FIELD field_333 blockId I
 	METHOD <init> (II)V
+		ARG 1 i
```
Comment (sschr15) [inline mappings/net/minecraft/item/Seeds.mapping]: What does this `i` do in the function body?

File: mappings/net/minecraft/storage/PlayerInventory.mapping [hunk 8]
```diff
@@ -15,4 +15,6 @@ CLASS net/minecraft/class_153 net/minecraft/storage/PlayerInventory
 		ARG 1 id
 			COMMENT The item id
 	METHOD method_385 getHeldItem ()Lnet/minecraft/class_127;
+	METHOD method_386 calculateDurability ()I
+	METHOD method_387 throwItems ()V
```
Comment (sschr15) [inline mappings/net/minecraft/storage/PlayerInventory.mapping]:
```suggestion
	METHOD method_387 dropItems ()V
```

File: mappings/net/minecraft/world/World.mapping [hunk 9]
```diff
@@ -14,6 +15,8 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 	FIELD field_299 diskSize J
 	FIELD field_300 illuminatedAreas Ljava/util/List;
 	FIELD field_301 tickableEntities Ljava/util/List;
+	FIELD field_302 treeSet Ljava/util/TreeSet;
```
Comment (sschr15) [inline mappings/net/minecraft/world/World.mapping]: Is this used anywhere? If so, its name should be more descriptive.

**Review by sschr15 — CHANGES_REQUESTED** [review]
Overall pretty good, though there are a couple changes I'd like to see.

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (tl0x) [reply mappings/net/minecraft/block/Block.mapping]: I made this change when I it was early in the day. In hindsight, I made many mistakes. I'll fix most of these tomorrow.

**Comment by tl0x** [comment]
Changes made.

File: mappings/net/minecraft/client/entity/particle/Particle.mapping [hunk 10]
```diff
@@ -17,5 +17,4 @@ CLASS net/minecraft/class_262 net/minecraft/client/entity/particle/Particle
 		ARG 12 velocityZ
 	METHOD method_650 render (Lnet/minecraft/class_188;FFFFFF)V
 		ARG 1 tesselator
-		ARG 2 g
```
Comment (tl0x) [inline mappings/net/minecraft/client/entity/particle/Particle.mapping]: adding this was a mistake when trying to edit another arg

File: mappings/net/minecraft/world/World.mapping [hunk 11]
```diff
@@ -14,6 +15,8 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 	FIELD field_299 diskSize J
 	FIELD field_300 illuminatedAreas Ljava/util/List;
 	FIELD field_301 tickableEntities Ljava/util/List;
+	FIELD field_302 worldTreeSet Ljava/util/TreeSet;
```
Comment (sschr15) [inline mappings/net/minecraft/world/World.mapping]: This still isn't very descriptive, I think I need to see what you're seeing in the bytecode, but I likely am unable to access until tonight.

File: mappings/net/minecraft/world/World.mapping [hunk 12]
```diff
@@ -15,7 +15,7 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 	FIELD field_299 diskSize J
 	FIELD field_300 illuminatedAreas Ljava/util/List;
 	FIELD field_301 tickableEntities Ljava/util/List;
-	FIELD field_302 treeSet Ljava/util/TreeSet;
+	FIELD field_302 worldTreeSet Ljava/util/TreeSet;
```
Comment (PhoenixVX) [inline mappings/net/minecraft/world/World.mapping]: What does this tree set manage though? Does it manage ex: removed entities? worldTreeSet isn't very descriptive of what it does functionally.

File: mappings/net/minecraft/world/World.mapping [hunk 13]
```diff
@@ -101,13 +109,31 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
...
+		ARG 1 pos1
+		ARG 2 pos2
+	METHOD method_234 getTargetFromVec3d (Lnet/minecraft/class_155;Lnet/minecraft/class_155;Z)Lnet/minecraft/class_157;
+		ARG 1 pos1
+		ARG 2 pos2
+	METHOD method_236 getBlocksInBox (Lnet/minecraft/class_156;)Ljava/util/List;
+		ARG 1 box
 	METHOD method_240 recursivelyDelete ([Ljava/io/File;)V
 		ARG 0 filesInFolder
+	METHOD method_241 shouldBeIgnited ()Z
```
Comment (sschr15) [inline mappings/net/minecraft/world/World.mapping:137]: I just realized: would it make more or less sense if this would be `shouldIgnite`?

File: mappings/net/minecraft/entity/living/LivingEntity.mapping [hunk 14]
```diff
@@ -22,6 +22,5 @@ CLASS net/minecraft/class_154 net/minecraft/entity/living/LivingEntity
 	METHOD method_393 onDeath (Lnet/minecraft/class_140;)V
 		ARG 1 killer
 	METHOD method_394 livingTick ()V
-	METHOD method_395 returnNullString ()Ljava/lang/String;
```
Comment (PhoenixVX) [inline mappings/net/minecraft/entity/living/LivingEntity.mapping]: Same as what sschr said. What does this do, and where is it used?

File: mappings/net/minecraft/block/tile/Furnace.mapping [hunk 15]
```diff
@@ -12,3 +12,4 @@ CLASS net/minecraft/class_36 net/minecraft/block/tile/Furnace
 	METHOD method_79 getSmeltedId (I)I
 		ARG 0 originalId
 	METHOD method_80 isBurning ()Z
+	METHOD method_81 continueSmelting ()Z
```
Comment (PhoenixVX) [inline mappings/net/minecraft/block/tile/Furnace.mapping]: Hmm...? This methods returns a boolean. I think this should be `shouldContinueSmelting`.

File: mappings/net/minecraft/block/TransparentBlock.mapping [hunk 16]
```diff
@@ -1,5 +1,4 @@
 CLASS net/minecraft/class_46 net/minecraft/block/TransparentBlock
-	FIELD field_178 transparent Z
```
Comment (PhoenixVX) [inline mappings/net/minecraft/block/TransparentBlock.mapping]: This is a boolean, it should be isTransparent.

File: mappings/net/minecraft/block/Block.mapping [hunk 17]
```diff
@@ -178,7 +178,6 @@ CLASS net/minecraft/class_39 net/minecraft/block/Block
 		ARG 3 y
 		ARG 4 z
 		ARG 5 face
-	METHOD method_121 returnZero ()I
```
Comment (PhoenixVX) [inline mappings/net/minecraft/block/Block.mapping]: Same as sschr.

File: build.gradle [hunk 18]
```diff
@@ -38,8 +38,6 @@ def build_version = System.getenv("MAVEN_PUSH_BRANCH") == "snapshots" ? String.v
 
 def yarnVersion = "${minecraft_version}+build${build_version}"
 
-println yarnVersion
```
Comment (PhoenixVX) [inline build.gradle]: Why is this here? It's unnecessary.

**Review by PhoenixVX — COMMENTED** [review]
Things

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (sschr15) [reply mappings/net/minecraft/block/Block.mapping]: `getLiquidType` or similar, perhaps? It's only implemented differently for water, and it returns `1` in that scenario.

File: mappings/net/minecraft/entity/living/LivingEntity.mapping [hunk 19]
```diff
@@ -7,15 +8,21 @@ CLASS net/minecraft/class_154 net/minecraft/entity/living/LivingEntity
...
+		ARG 2 y
+		ARG 3 z
 	METHOD method_390 heal (I)V
 		ARG 1 amount
 	METHOD method_391 getDropItem ()I
 	METHOD method_392 updateMovements ()V
 	METHOD method_393 onDeath (Lnet/minecraft/class_140;)V
 		ARG 1 killer
 	METHOD method_394 livingTick ()V
+	METHOD method_395 getAmbientSound ()Ljava/lang/String;
```
Comment (sschr15) [inline mappings/net/minecraft/entity/living/LivingEntity.mapping]: Your `getIdleSound` is better, pls commit this and the tick variable with your better name thanks

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (tl0x) [reply mappings/net/minecraft/block/Block.mapping]: I'm going to remove the mapping until I figure out what it actually does.

**Review by sschr15 — CHANGES_REQUESTED** [review]
All good, except for the `worldTreeSet` field which should probably get a more descriptive name.

**Comment by tl0x** [comment]
I gave it a more descriptive name after some testing.

**Review by sschr15 — APPROVED** [review]
